AdobeIms/Controller/Adminhtml/User/Profile.php

```diff
@@ -10,14 +10,15 @@
 use Magento\AdobeImsApi\Api\UserProfileRepositoryInterface;
 use Magento\Authorization\Model\UserContextInterface;
 use Magento\Backend\App\Action;
+use Magento\Framework\App\Action\HttpPostActionInterface;
 use Magento\Framework\Controller\ResultFactory;
 use Magento\Framework\Exception\NoSuchEntityException;
 use Psr\Log\LoggerInterface;
 
 /**
  * Get Adobe services user account action
  */
-class Profile extends Action
+class Profile extends Action implements HttpPostActionInterface
```

Comment:
This controller is used to retrieve data only, let's make it implement the Get interface and change the call in `.modman/adobe-stock-integration/AdobeIms/view/adminhtml/web/js/signIn.js`